packages/atlas/src/components/_buttons/GoogleButton/GoogleButton.tsx

```diff
@@ -0,0 +1,61 @@
+import styled from '@emotion/styled'
+import * as React from 'react'
+
+import { SvgGoogleSmallLogo } from '@/assets/icons/GoogleSmallLogo'
+
+interface GoogleButtonProps extends React.HTMLAttributes<HTMLButtonElement> {
+  label?: string
+  disabled?: boolean
+}
+export const GoogleButton = ({ label, ...rest }: GoogleButtonProps) => {
+  return (
+    <Container as="button" {...rest}>
```

Comment:
This is a button, we don't need `as` here
```suggestion
    <Container {...rest}>
```